python/py-fastapi/Portfile

```
@@ -21,9 +21,9 @@ long_description    FastAPI is a modern, fast (high-performance), web \
                     framework for building APIs with Python 3.6+ based \
                     on standard Python type hints.
 
-checksums           rmd160  25c8572c976bf8e1cd043a2275809017c6e82f3b \
-                    sha256  f6e9bcc0a8431a6dca4efafc636c5082dc0dd32798c8495d6280452dbc70208e \
-                    size    5270757
+checksums           rmd160  63fc2a29ad960f4ad23cdcf745c62d3310beace4 \
+                    sha256  9644365c0a9fd861e9fdc8c77efda41b55ef4c2aa77bbd661d0a903037cfb6f9 \
+                    size    6808068
 
 python.versions     37 38 39
```

Python 3.10 is the current default Python - perhaps we should take the opportunity to add the subport for 3.10 as well:

```suggestion
python.versions     37 38 39 310
```

---

In this case the linter and installer do not recognize the dependency:

```
--->  Verifying Portfile for py-fastapi
Error: Unknown dependency: py310-fastapi
--->  1 errors and 0 warnings found.
--->  Computing dependencies for py-fastapi.
Error: Dependency 'py310-fastapi' not found.
Error: Follow https://guide.macports.org/#project.tickets if you believe there
is a bug.
Error: Processing of port py-fastapi failed
```
This happens even with ports that have dependencies updated to Python 3.10.